### PR TITLE
Fix pylint warnings in gprecoverseg

### PR DESCRIPTION
**Pull request by royc1**
There is no functional changes in this commit. Fixes pylint warnings and whitespace.

Authors: Chris Hajas and Chumki Roy

### CODE REVIEW

File: gpMgmt/bin/gppylib/programs/clsRecoverSegment.py [hunk 1]
```diff
@@ -354,7 +354,7 @@ def __readSpareDirectoryMap(self, gpArray, spareDataDirectoryFile):
                             (len(fsOidToPath), len(filespaceNameToFilespace)))
         return fsOidToPath
 
-    def __applySpareDirectoryMapToSegment( self, gpEnv, gpArray, spareDirectoryMap, segment):
```
Comment (khuddlefish) [inline gpMgmt/bin/gppylib/programs/clsRecoverSegment.py]: What's the difference between gpArray and gparray? We call the latter in this function, but are we sure that's the correct one?

File: gpMgmt/bin/gppylib/programs/clsRecoverSegment.py [hunk 2]
```diff
@@ -711,15 +711,8 @@ def SanFailback(self, array_config, gpEnv):
...
-        type = mp_dict['type']
-
-        if active == 'm':
-            oldhost = mp_dict['primaryhost']
-            old_mp = mp_dict['primarymountpoint']
 
-            newhost = mp_dict['mirrorhost']
-            new_mp = mp_dict['mirrormountpoint']
-        else:
+        if active !='m':
```
Comment (Chibin) [inline gpMgmt/bin/gppylib/programs/clsRecoverSegment.py]: need a space `active != 'm':`

File: gpMgmt/bin/gppylib/programs/clsRecoverSegment.py [hunk 3]
```diff
@@ -354,7 +354,7 @@ def __readSpareDirectoryMap(self, gpArray, spareDataDirectoryFile):
                             (len(fsOidToPath), len(filespaceNameToFilespace)))
         return fsOidToPath
 
-    def __applySpareDirectoryMapToSegment( self, gpEnv, gpArray, spareDirectoryMap, segment):
+    def __applySpareDirectoryMapToSegment( self, gpEnv, spareDirectoryMap, segment):
```
Comment (Chibin) [inline gpMgmt/bin/gppylib/programs/clsRecoverSegment.py]: `( self` to `(self`

File: gpMgmt/bin/gppylib/programs/clsRecoverSegment.py [hunk 4]
```diff
@@ -490,7 +490,7 @@ def getRecoveryActionsFromConfiguration(self, gpEnv, gpArray):
                 # these two lines make it so that failoverSegment points to the object that is registered in gparray
                 failoverSegment = failedSegment
                 failedSegment = failoverSegment.copy()
-                self.__applySpareDirectoryMapToSegment( gpEnv, gpArray, spareDirectoryMap, failoverSegment)
+                self.__applySpareDirectoryMapToSegment( gpEnv, spareDirectoryMap, failoverSegment)
```
Comment (Chibin) [inline gpMgmt/bin/gppylib/programs/clsRecoverSegment.py]: `( gpEnv` to `(gpEnv`

File: gpMgmt/bin/gppylib/programs/clsRecoverSegment.py (hunk 3 above)
Comment (royc1) [inline gpMgmt/bin/gppylib/programs/clsRecoverSegment.py]: Fixed thanks!

**Pull request merged**